api-module-library/sharepoint/defaultConfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "microsoft-sharepoint",
+    "label": "Microsoft SharePoint",
+    "productUrl": "https://microsoft.com/sharepoint",
+    "apiDocs": "Good Luck",
+    "logoUrl": "https://friggframework.org/assets/img/microsoft-sharepoint.jpeg",
+    "categories": ["Sharing"],
+    "description": "If you know, you know"
```

Comment:
Can you grab Microsoft's description of SharePoint from somewhere? Maybe the H1 of the product site? One or two sentences is fine